src/helpers/handlebars.ts

```diff
@@ -47,4 +48,13 @@ export const repoName = (v1: string): string => {
 
 export const addClassIfEqual = (v1: number, v2: number, extra: number, v3: string): string => {
   return v1 === (v2+extra) ? v3 : '';
+};
+
+export const removeDomain = (fullUrl: string): string => {
```

Comment:
I don't think that we need to add a helper for it.

This `fullUrl` is declared at [/src/index.ts#L66](https://github.com/osscameroon/miniyotas/blob/7610ee6837aac63d76f4483f48891a8cab528be5/src/index.ts#L66), and looks used only in this `pagination.hbs` file.